mage_ai/data_preparation/shared/secrets.py

```diff
@@ -72,20 +75,20 @@ def get_valid_secrets() -> List:
     return valid_secrets
 
 
-def get_secret_value(name: str, repo_name: str = None) -> str:
+def get_secret_value(name: str, repo_name: str = get_repo_path()) -> str:
```

Comment:
It'll be better to set the default value to None in method definition. Then inside the method, if repo_name is None, repo_name = get_repo_path().

When the method is defined, repo path might not be set yet.